libs/utils/utils.js

```diff
@@ -109,13 +183,24 @@ export async function loadBlock(block) {
   return block;
 }
 
-export async function loadLCP({ blocks = [], lcpList = LCP_BLOCKS, loader = loadBlock }) {
+export async function loadLCP({ blocks = [], lcpList = LCP_BLOCKS }) {
   const lcpBlock = blocks.find((block) => lcpList.includes(block.classList[0]));
   if (lcpBlock) {
     const lcpIdx = blocks.indexOf(lcpBlock);
     blocks.splice(lcpIdx, 1);
-    await loader(lcpBlock, true);
+    await loadBlock(lcpBlock, true);
   }
+  const lcpImg = document.querySelector('main img');
+  return new Promise((resolve) => {
+    if (lcpImg && !lcpImg.complete) {
+      /* c8 ignore next 3 */
+      lcpImg.setAttribute('loading', 'eager');
+      lcpImg.addEventListener('load', () => resolve(lcpImg));
```

Comment:
prob doesn't matter but should these event listeners be removed after load/error?

Reply:
That's _a lot_ (IMHO) of extra code to change. We'd have to make new functions to be able to remove them.